### PR TITLE
Created e2e tests

### PR DESCRIPTION
**Pull request by timeab**
I’ve created a new folder e2e where we have the selectors in a separate folder because I’ve locate all the commonSelectors from the app. I used get methods where I just return the selector. Also I’ve implemented test’s like:
- Check description from the first selected option,
- Check addToCart functionality,
- Check all checkbox items, 
- Uncheck all checkbox items,
- Check if selecting a category contains the right number of description items.

### CODE REVIEW

File: e2e/firstTest.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import commonSelectors from './selectors/commonSelectors';
+import descriptionContent from './selectors/descriptionContent';
+
+const commonSelector = new commonSelectors;
+const categoryDescriptionItems = { Kids: 1, Home: 5, Auto: 5, Food: 1 };
+
+fixture`Testing product app`// declare the fixture
+    .page`http://localhost:3000/`;
```
Comment (kovacsalexandrurobert) [inline e2e/firstTest.js]: can you move it to a config? .env file?

File: e2e/firstTest.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            .expect(Boolean(commonSelector.getAllCheckBoxes().nth(0).child().value)).eql(false);
+    });
+});
+test('Check if selecting a category contains the right number of description items', async t => {
+    await Promise.all(Object.entries(categoryDescriptionItems).map(async (items, index) => {
+        const [option, descriptionLength] = [...items];
+        const cardListItems = commonSelector.getAllCardList().child();
+        const checkBoxItem = commonSelector.getAllCheckBoxes().nth(index).withText(option);
+        await t.click(checkBoxItem)
+            .expect(cardListItems.find('.description').count).eql(descriptionLength)
```
Comment (kovacsalexandrurobert) [inline e2e/firstTest.js:45]: the find(), you can move it to a const.. adds to readability

File: e2e/firstTest.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+});
+test('Check all checkbox items', async t => {
+    Object.keys(categoryDescriptionItems).map(async option => {
+        await t.click(commonSelector.getAllCheckBoxes().withText(option))
+            .expect(Boolean(commonSelector.getAllCheckBoxes().nth(0).child().value)).eql(true);
+    });
+});
+test('Uncheck all checkbox items', async t => {
+    Object.keys(categoryDescriptionItems).map(async option => {
+        await t.click(commonSelector.getAllCheckBoxes().withText(option))
```
Comment (kovacsalexandrurobert) [inline e2e/firstTest.js]: commonSelector.getAllCheckBoxes().withText(option) is it ok to move it to a const? you are using it everywhere

**Review by kovacsalexandrurobert — CHANGES_REQUESTED** [review]

File: e2e/firstTest.js (hunk 3 above)
Comment (timeab) [reply e2e/firstTest.js]: Done

File: e2e/firstTest.js (hunk 2 above)
Comment (timeab) [reply e2e/firstTest.js:45]: Done

File: e2e/firstTest.js (hunk 1 above)
Comment (timeab) [reply e2e/firstTest.js]: Done

**Review by kovacsalexandrurobert — APPROVED** [review]

**Pull request merged**